website/pages/blog/product-updates-8.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: "CloudQuery Product Updates #8"
+tag: product
+date: 2023/09/28
+description: >-
+  Monthly updates on CloudQuery product and roadmap.
+author: michal
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+Hey everyone! Our monthly round-up of CloudQuery product and roadmap updates is here!
+
+TLDR:
+
+- Open source: A new SDK, new plugins, performance improvements
+- Premium (closed source): New policies
+
+## SDKs!
+
+ICYMI, we published [Python](https://www.cloudquery.io/blog/announcing-cloudquery-python-sdk) and [JavaScript](https://www.cloudquery.io/blog/announcing-cloudquery-javascript-sdk) SDKs last month and this month, we followed with Java!
+
+- Check out our [Java announcement blog](https://www.cloudquery.io/blog/announcing-cloudquery-java-sdk)!
+- Creating a [Java source](https://www.cloudquery.io/docs/developers/creating-new-plugin/java-source)
+
+## Sources
+
+### New Source Plugins
+
+- [Bitbucket](https://www.cloudquery.io/docs/plugins/sources/bitbucket/overview) - First plugin utilizing our Java SDK!
+- [Notion](https://github.com/cloudquery/cloudquery/tree/main/plugins/source/notion) - Loads databases, pages, and users from Notion to any database, data warehouse or data lake supported by CloudQuery, such as PostgreSQL, BigQuery, Athena, and many more.
+- You can now configure the API version on the [Shopify](https://www.cloudquery.io/docs/plugins/sources/shopify/overview) plugin.
```

Comment:
This is a nit, does configuring the API version fall under "new source plugins"?